CHANGELOG.md

```diff
@@ -103,6 +103,14 @@
   when calling `NSNumber.init(value:)` directly.  
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#5172](https://github.com/realm/SwiftLint/issues/5172)
+* The `no_magic_numbers` rule will not trigger for the second operand of
```

Comment:
Guessing a merge issue? I think the one at line 110 is the best...

Reply:
nice spot! Thanks!